File: catkin_ws/src/Task2/scripts/increasing.py
#!/usr/bin/env python
import rospy
from std_msgs.msg import Int32


inc = 0
rospy.init_node("increasing")
rate = rospy.Rate(1)
publisher = rospy.Publisher("/inc", Int32)

while not rospy.is_shutdown():
    publisher.publish(inc)
    inc += 1
    rate.sleep()

File: catkin_ws/src/Task2/src/increasing.cpp
#include <ros/ros.h>
#include <std_msgs/Int32.h>

int main(int argc, char **argv) {
    ros::init(argc, argv, "increase");
    ros::NodeHandle node;
    ros::Publisher pub = node.advertise<std_msgs::Int32>("inc", 10);
    ros::Rate rate(1);
    int count = 0;
    std_msgs::Int32 msg;

    while (ros::ok()) {
        msg.data = count;
        pub.publish(msg);
        ++count;
        rate.sleep();
    }
    return 0;
}